### PR TITLE
WIP - Split blobstore cassandra implementation

### PR DESCRIPTION
**Pull request by remk**

### CODE REVIEW

**Comment by Arsnael** [comment]
Ticket number on those commits please...

File: server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/CassandraBlobStore.java [hunk 1]
```diff
@@ -19,203 +19,127 @@
...
+        this.dumbBlobStore = dumbBlobStore;
     }
 
     @VisibleForTesting
-    public CassandraBlobStore(Session session) {
-        this(new CassandraDefaultBucketDAO(session),
-            new CassandraBucketDAO(new HashBlobId.Factory(), session),
-            CassandraConfiguration.DEFAULT_CONFIGURATION,
-            new HashBlobId.Factory());
+    public static CassandraBlobStore forTesting(Session session) {
```
Comment (Arsnael) [inline server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/CassandraBlobStore.java]: Is it ok to declare testing static methods in a main class like that?

Comment (chibenwa) [reply server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/CassandraBlobStore.java]: I personnaly concider it OK as the intent in the naming is explicit.

**Comment by remk** [comment]
test this please

**Comment by remk** [comment]
replaced by :  https://github.com/linagora/james-project/pull/3051

**Pull request closed without merging**